DMCompiler/Compiler/DM/DMParser.cs

```diff
@@ -1340,7 +1340,7 @@ public partial class DMParser : Parser<Token> {
                     tryBody = new DMASTProcBlockInner(loc,statement);
                 }
 
-                Newline();
+                BracketWhitespace();
```

Comment:
`BracketWhitespace()` is meant to only be used inside `( ... )` or `[ ... ]`. I'd prefer if you instead put `Whitespace()` after the `Newline()` call

```suggestion
                Newline();
                Whitespace();
```

I intend to remove `BracketWhitespace()` at some point by making the lexer instead ignore whitespace inside brackets.